src/views/PurchaseOrderDetail.vue

```diff
@@ -204,6 +253,20 @@ export default defineComponent({
       if(this.queryString) payload = this.queryString
       this.store.dispatch('order/updateProductCount', payload)
     },
+    getPOItemsCount(orderType: string) {
```

Comment:
We can use a single method for getting the count and items and use `.length` in the template to get the count